docs/deployment/file-access-auth.md

```diff
@@ -0,0 +1,159 @@
+---
+title: File Access Authorization
+parent: Deployment
+nav_order: 4
+---
+
+# (BETA) File Access Authorization
+
+> **IMPORTANT:** This feature is currently in beta. Configuration and log formats are subject to change.
```

Comment:
```suggestion
> **IMPORTANT:** This feature is in beta. Configuration and log formats are subject to change.
```

Reply:
Perhaps remove "currently" it doesn't really add anything. The beta label will be removed once the feature development is at a particular point anyway. 

In the note, "in beta" is fine, but another alternative is:

> **IMPORTANT:** This feature is in development. Configuration and log formats are subject to change.

"development" feels more active and provides more context when used with the "(BETA)" label. But, totally up to you, either way is good.